README.md

```diff
@@ -854,6 +854,43 @@
         .call(tron.led);
     ```
 
+  - Leave a blank line after blocks and before the next statement
+
+    ```javascript
+    // bad
+    if (true) {
+      var a = 2;
+    }
+    var b = 3;
+
+    // good
+    if (true) {
+      var a = 2;
```

Comment:
I personally prefer to separate logical blocks with an extra line and functions with two. So I would support this in general. The only issue is to figure out what that logical block is and how to generalize the recommendation.
I also consider this to be very minor and non-strict recommendation, so never raise it on code reviews. Unless it is a very bad case.